storage/mysql/tree_storage.go

```diff
@@ -60,11 +60,6 @@ func openDB(dbURL string) (*sql.DB, error) {
 		return nil, err
 	}
 
-	if _, err := db.Exec("SET sql_mode = 'STRICT_ALL_TABLES'"); err != nil {
```

Comment:
Not sure why this was removed, doesn't immediately seem related. Is there some problem with it enabled now?